### ISSUE TITLE
017 copyright or legal deposit number

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#55]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/0XX.csv

**Comment by lake44me** [comment]
$2 - Source (NR)
Code from: [Copyright and Legal Deposit Number Source Codes](https://www.loc.gov/standards/sourcelist/copyright-legal-deposit.html).

This code list is not in id.loc.gov. There is only one source code in the document:
Copyright and Legal Deposit Number Source Codes
   rocgpt
   R.O.C. Government Publications Catalogue (Taipei: Research, Development and Evaluation Commission, Executive Yuan)

So effectively it appears LC abandoned trying to build a set of source codes for this identifier.

**Comment by lake44me** [comment]
Reviewer:  I chose to use Nomen property has context of use http://rdaregistry.info/Elements/n/P80067 with value "Copyright or legal deposit number".  There are other possibilities - if there's a reason to use category of nomen or something else please advise

**Comment by sarahruska** [comment]
Hi @lake44me - I wanted to doublecheck on $d being deleted? In the [MARC bibliographic](https://www.loc.gov/marc/bibliographic/bd017.html) it looks like it's still valid. Wasn't sure what I was missing!

**Comment by lake44me** [comment]
@sarahruska good catch.  I omitted $d from the group of subfields in the first "first pass" row . It's in there in the transformation notes, but not in the Recording Method list of properties either.  I updated that line, so it's ok to delete the separate line for $d

I am choosing not to map that subfield for the second first pass line - the one for the invalid number in $z; in case both a valid and invalid number appear in the same MARC tag with a $d, the date of usage should map with the valid one, and I doubt it will be found when an 017 only has an invalid number (and who cares, anyway...).  Enough!

**Comment by sarahruska** [comment]
Thanks, Laura!
